src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/ConversionOperatorExtensions.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    internal static class ConversionOperatorExtensions
+    {
+        public static SpecialType[]? GetBuiltInNumericConversions(this ITypeSymbol container)
+        {
+            // Source: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/conversions#explicit-numeric-conversions
+            return container.SpecialType switch
```

Comment:
could use https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs#L1743 for efficiency.

Reply:
Maybe. But `ConversionsBase.s_explicitNumericConversions` is `private` and so are all of it's usages. I doubt I can use it directly from here. So I probably need to copy it. I also don't see how the bool array helps here as I need to return a `SpecialType[]`. So some kind of mapping needs to be done anyway.

Reply:
I didn't mean to use directly. the same mechanism can be duplicated here.